src/compiler/types.ts

```diff
@@ -7098,7 +7008,10 @@ export interface ProjectReference {
     path: string;
     /** The path as the user originally wrote it */
     originalPath?: string;
-    /** True if the output of this reference should be prepended to the output of this project. Only valid for --outFile compilations */
+    /**
+     * Only present for deprecate notification
+     * @internal
+     */
     prepend?: boolean;

```

Comment:
In #57527 I just marked "removed" things as deprecated but left them in, which has the benefit of letting people still compile (and us getting a strikethrough), and then in 6.0 we could just delete them wholesale. I feel like that's safer, but I'm not 100% certain we've actually talked about how we want to do this sort of thing.

Reply:
They have been getting strike through for since 5.0 no ?

Reply:
oh this one we forgot to deprecate before so makes sense to mark deprecate.

Reply:
Yeah, we didn't do that :(